src/aposteriori/data_prep/create_frame_data_set.py

```diff
@@ -752,6 +752,29 @@ def process_paths(
     return
 
 
+def _select_pdb_chain(pdb_path: pathlib.Path, chain: str):
```

Comment:
I think you need a bit clearer documentation here to say that it's overwriting the file. You want to make sure that side effects (anything that's changed but not directly returned by the function _i.e._ changing global variables, IO, http requests) are really well labelled in you doc string, as it's difficult behaviour to debug.

Reply:
Yep that's a really good point. Do you think it should throw a warning too?

Reply:
No I don't think so, but see my other comment because I think there's a safer way to do this.